test/test_randomforest.jl

```diff
@@ -32,6 +32,15 @@ using DataFrames
     accuracy = sum(predictions .== test_labels) / length(test_labels)
 
 
+    @test Set(predictions) <= Set(test_labels)
```

Comment:
I added this test to cover the line that was uncovered because i fixed the error in the other test ^^